### PR TITLE
docs: fix the setup workflow #591

### PR DESCRIPTION
**Pull request by EvsChen**
**What kind of change does this PR introduce?**
Improve docs

**Summary**
As mentioned in #591 , the setup guidance is not clear enough for newcomers who setup the directory for the first time. The bootstrap command is necessary.

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ In case you are suggesting a new feature, we will match your idea with our curre
 
 ### Setup with npm
 * Install the dependencies: `npm install`
+* Bootstrap the packages:`npx lerna bootstrap`
```
Comment (ev1stensberg) [inline CONTRIBUTING.md]: No need to use npx, regular npm is sufficient

Comment (EvsChen) [reply CONTRIBUTING.md]: You mean `npm lerna bootstrap`? It doesn't seem to work for me. I use npm@6.4.1

Comment (ev1stensberg) [reply CONTRIBUTING.md]: Npm run bootstrap is the command you’re looking for I assume

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -42,6 +43,7 @@ In case you are suggesting a new feature, we will match your idea with our curre
 
 ```bash
 yarn install
+yarn lerna bootstrap
```
Comment (ev1stensberg) [inline CONTRIBUTING.md]: Yarn bootstrap

File: CONTRIBUTING.md (hunk 1 above)
Comment (EvsChen) [reply CONTRIBUTING.md]: Yes, you're right. My initial thought was to keep consistent between yarn and npm.

**Comment by EvsChen** [comment]
Sorry I'm a little confused. I think it's either `npm run bootstrap` or  `npm install && npx lerna bootstrap`, since the bootstrap command runs clean all. The same for yarn.

**Comment by ev1stensberg** [comment]
You need to install deps first. Add those lines and youre good. Npm install and yarn

**Comment by EvsChen** [comment]
But there is a `npm install` in the bootstrap script.

**Pull request merged**